corehq/apps/app_manager/static/app_manager/js/releases.js

```diff
@@ -258,24 +258,15 @@ hqDefine('app_manager/js/releases.js', function () {
             }
         });
 
-        self.previousReleaseId = function(index) {
-            var savedApp;
-            for (var i = index + 1; i < self.savedApps().length; i++) {
-                savedApp = self.savedApps()[i];
-                if (savedApp.is_released()) {
-                    return savedApp.id();
-                }
+        self.previousBuildId = function(index) {
+            if (self.savedApps()[index + 1]) {
```

Comment:
this was just dumb. i was comparing the latest starred version instead of the latest builds